compiler-core/src/config.rs

```diff
@@ -713,6 +713,8 @@ pub struct DenoConfig {
     pub allow_all: bool,
     #[serde(default)]
     pub unstable: bool,
+    #[serde(default)]
+    pub location: String,
```

Comment:
How about we make this a `Uri`?

Reply:
Good call, I'll update to use the `Uri` type now 👍🏻